build.xml

```diff
@@ -259,7 +263,7 @@ lib/plexi submodule or add the the command line argument
         <attribute name="Class-Path" value="${tmp.adaptor.classpath}"/>
         <section name="com/google/enterprise/adaptor/sharepoint/">
           <attribute name="Implementation-Title"
-            value="Google Adaptor for SharePoint"/>
+            value="${adaptor.title}"/>
```

Comment:
It looks like all three of these edited & added attribute elements will now fit on one line.